packages/components/package.json

```diff
@@ -62,7 +62,7 @@
 		"highlight-words-core": "^1.2.2",
 		"lodash": "^4.17.21",
 		"memize": "^1.1.0",
-		"moment": "^2.22.1",
+		"moment": "^2.26.0",
```

Comment:
Yeah, this was a whole thing. I think I had to update `@types/react-dates` so that I could get `tsc` to be happy about using `renderMonthElement` and then that newer version of `@types/react-dates` expected a newer version of `moment`.

Reply:
Got it, thank you!